docs/src/developer/introduction.md

```diff
@@ -55,17 +55,17 @@ In the `src` directory of Nemo are four directories `flint`, `arb`, `antic` and
 `test` directory is similarly organised.
 
 Within each of these directories is a set of files, one per module within
-the C libraries, e.g. the `fmpz.jl` file wraps the Flint `fmpz` module for
-multiple precision integers. The `fmpz_poly.jl` file wraps the Flint
+the C libraries, e.g. the `fmpz.jl` file wraps the FLINT `fmpz` module for
+multiple precision integers. The `fmpz_poly.jl` file wraps the FLINT
 univariate polynomials over `fmpz` integers, and so on.
 
-The `QQFieldElem` prefix is for Flint rationals, `FqPolyRepFieldElem` for Flint finite fields with
+The `QQFieldElem` prefix is for FLINT rationals, `FqPolyRepFieldElem` for FLINT finite fields with
```

Comment:
Something is wrong here (predating this PR), we should propose say fmpy? Maybe also elsewhere

Reply:
I don't follow you, can you explain what you mean?

Reply:
https://github.com/Nemocas/Nemo.jl/pull/1376 and https://github.com/Nemocas/Nemo.jl/pull/1638 changed the text in this block more or less by accident (large renaming script, and we didn't check all docs if they are still comprehensible. I'll fix that in https://github.com/Nemocas/Nemo.jl/pull/2003.